server/src/main/java/com/defold/extender/services/RealGradleService.java

```diff
@@ -280,4 +284,16 @@ private List<File> downloadDependencies(File cwd) throws IOException, ExtenderEx
         return unpackedDependencies;
     }
 
+    private void writeDependencyTree(File out, File cwd) throws IOException, ExtenderException {
+        long methodStart = System.currentTimeMillis();
+        LOGGER.info("Writing dependency tree");
+
+        String treelog = execCommand("gradle dependencies --configuration releaseCompileClasspath", cwd);
+        LOGGER.info("\n" + treelog);
```

Comment:
I guess it's worth mark it as `debug` level log because tree log can be huge.

Reply:
Good point.